<a href="https://colab.research.google.com/github/omkar87796-sudo/personal-diary/blob/main/rendom_sampling_without_replace.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
import pandas as pd
import numpy as np

we have 100 pepole salary data and we need all pepole avg salary

In [ ]:
data = np.random.randint(20000, 80000, 100)
data = pd.DataFrame(data)

we made this data using numpy random library

In [ ]:
data

0
0   75658
1   34303
2   65164
3   56142
4   64856
..    ...
95  34117
96  40405
97  64422
98  61566
99  49230

[100 rows x 1 columns]

we put the data in pandas library

In [ ]:
group1 = data[0:15]
group2 = data[15:30]
group3 = data[30:45]
group4 = data[45:60]
group5 = data[60:75]
group6 = data[75:90]
group7 = data[90:100]


divided in 7 part

In [ ]:
total_avrage  = (np.average(group1) + np.average(group2) + np.average(group3) + np.average(group4) + np.average(group5) + np.average(group6) + np.average(group7))/7
print(total_avrage)

51524.019047619055


we take a average of all parts and then add them and divided by 7 and aur final ans is " avrage salary = 51524.019047619055 "